<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>776</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>3</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>LogMessage</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTextEdit" name="textEdit_log"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget_1DFFT">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>9</horstretch>
        <verstretch>8</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>ADC-T2R2</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QWidget" name="widget_tx0rx1" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QWidget" name="widget_tx1rx0" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QWidget" name="widget_tx1rx1" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QWidget" name="widget_tx0rx0" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>ADC</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QWidget" name="widget_adc" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>1D_FFT</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QWidget" name="widget_1DFFT" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>UDP_config</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_Connect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Disconnect">
         <property name="text">
          <string>Disconnect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_IsSave">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Is_SaveMat</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="pushButton_ReadFile">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>ReadFile</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_MatFrom"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Next">
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_CloseFile">
         <property name="text">
          <string>CloseFile</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Connect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>UDP_connect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>911</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>927</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Disconnect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>UDP_disconnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>911</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>928</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ReadFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ReadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>853</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>806</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Next</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ShowNextFrame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>961</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>805</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_CloseFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>CloseFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>877</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>806</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>UDP_connect()</slot>
  <slot>UDP_disconnect()</slot>
  <slot>ReadFile()</slot>
  <slot>ShowNextFrame()</slot>
  <slot>CloseFile()</slot>
 </slots>
</ui>
